src/activity_actor.cpp

```diff
@@ -5716,7 +5716,7 @@ void firstaid_activity_actor::serialize( JsonOut &jsout ) const
 
 std::unique_ptr<activity_actor> firstaid_activity_actor::deserialize( JsonValue &jsin )
 {
-    firstaid_activity_actor actor( {} );
+    firstaid_activity_actor actor( {}, {} );
```

Comment:
Is this accidental change?

Reply:
No, for consistency, I wanted to have each deserialize method call the same constructor variant.